resources/todomvc/readme.md

```diff
@@ -1,142 +1,30 @@
-Addy is a Developer Platform Engineer at Google who originally created TodoMVC. He oversees the project direction, drives expansion and helps lead core development with Sindre Sorhus (by far our most active contributor!).
-
-#### [Sindre Sorhus](https://github.com/sindresorhus) - Lead Developer
-
-<img align="left" width="40" height="40" src="http://www.gravatar.com/avatar/d36a92237c75c5337c17b60d90686bf9.png?s=40">
-Sindre is a Web Developer who drives core development, quality control and application design for the project. His contributions have helped us ensure consistency and best practices are enforced wherever possible.
-
-#### [Pascal Hartig](https://github.com/passy) - Developer
-
-<img align="left" width="40" height="40" src="http://www.gravatar.com/avatar/be451fcdbf0e5ff07f23ed16cb5c90a3.png?s=40">
-Pascal is a Software Engineer at weluse GmbH with a deep passion for consistency. He watches pull requests and helps developers getting their contributions integrated with TodoMVC.
-
-#### [Stephen Sawchuk](https://github.com/stephenplusplus) - Developer
-
-<img align="left" width="40" height="40" src="https://secure.gravatar.com/avatar/098cfe2d360e77c3229f2cd5298354c4?s=40">
-Stephen is a Front-end Engineer at Quicken Loans that cares about improving the maintainability and developer experience of open-source projects. His recent contributions include helping us move all apps over to using Bower and implementing the new information bar.
-
-#### [Gianni Chiappetta](https://github.com/gf3) - Logo designer
-
-<img align="left" width="40" height="40" src="http://www.gravatar.com/avatar/4b0209ae3652cc5a7d53545e759fbe39.png?s=40">
-Gianni is a programmer and designer currently working as the Chief Rigger at MetaLab.
-
-## Disclaimer
-
-<img align="right" width="230" height="230" src="https://raw.github.com/tastejs/todomvc/gh-pages/media/icon-small.png">
-
-TodoMVC has been called many things including the 'Speed-dating' and 'Rosetta Stone' of MV* frameworks. Whilst we hope that this project is able to offer assistance in deciding what frameworks are worth spending more time looking at, remember that the Todo application offers a limited view of what a framework may be capable of.
-
-It is meant to be used as a gateway to reviewing how a basic application using a framework may be structured and we heavily recommend investing time researching a solution in more depth before opting to use it.
-
-
-## Getting Involved
+![screenshot](./screenshot.png)
 
-Whilst we enjoy implementing and improving existing Todo apps, we're always interested in speaking to framework authors (and users) wishing to share Todo app implementations in their framework/solution of choice.
+## What are we testing
 
-Check out our [contribution docs](contributing.md) for more info.
+- Stress-test DOM manipulations of a repeated action.
+- Impact of JavaScript version releases.
```

Comment:
What is this referring to? I'm guessing language features, like the VanillaJS-ES2015 test?

Reply:
yes, that's correct - I can add more clarity